Index: libX11-X11R7.0-1.0.0/x11.pc.in
===================================================================
--- libX11-X11R7.0-1.0.0.orig/x11.pc.in	2006-02-23 23:12:57.000000000 -0500
+++ libX11-X11R7.0-1.0.0/x11.pc.in	2006-02-23 23:20:32.000000000 -0500
@@ -8,7 +8,7 @@
 Name: X11
 Description: X Library
 Version: @PACKAGE_VERSION@
-Requires: xproto @XKBPROTO_REQUIRES@
+Requires: xproto, kbproto
 Requires.private: xau xdmcp
 Cflags: -I${includedir} @XTHREAD_CFLAGS@
 Libs: -L${libdir} -lX11
