Index: libX11-X11R7.0-1.0.0/x11.pc.in
===================================================================
--- libX11-X11R7.0-1.0.0.orig/x11.pc.in	2006-02-25 01:11:37.000000000 +0700
+++ libX11-X11R7.0-1.0.0/x11.pc.in	2006-02-25 01:17:12.000000000 +0700
@@ -8,7 +8,7 @@
 Name: X11
 Description: X Library
 Version: @PACKAGE_VERSION@
-Requires: xproto @XKBPROTO_REQUIRES@
+Requires: xproto kbproto
 Requires.private: xau xdmcp
 Cflags: -I${includedir} @XTHREAD_CFLAGS@
 Libs: -L${libdir} -lX11
