diff --git a/man/AllPlanes.man b/man/AllPlanes.man
index d5fbb6f..829a45d 100644
--- a/man/AllPlanes.man
+++ b/man/AllPlanes.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH AllPlanes 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH AllPlanes __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 AllPlanes, BlackPixel, WhitePixel, ConnectionNumber, DefaultColormap, DefaultDepth, XListDepths, DefaultGC, DefaultRootWindow, DefaultScreenOfDisplay, DefaultScreen, DefaultVisual, DisplayCells, DisplayPlanes, DisplayString, XMaxRequestSize, XExtendedMaxRequestSize, LastKnownRequestProcessed, NextRequest, ProtocolVersion, ProtocolRevision, QLength, RootWindow, ScreenCount, ScreenOfDisplay, ServerVendor, VendorRelease \- Display macros and functions
 .SH SYNTAX
diff --git a/man/BlackPixelOfScreen.man b/man/BlackPixelOfScreen.man
index 08c3cd5..68f38ed 100644
--- a/man/BlackPixelOfScreen.man
+++ b/man/BlackPixelOfScreen.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH BlackPixelOfScreen 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH BlackPixelOfScreen __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 BlackPixelOfScreen, WhitePixelOfScreen, CellsOfScreen, DefaultColormapOfScreen, DefaultDepthOfScreen, DefaultGCOfScreen, DefaultVisualOfScreen, DoesBackingStore, DoesSaveUnders, DisplayOfScreen, XScreenNumberOfScreen, EventMaskOfScreen, HeightOfScreen, HeightMMOfScreen, MaxCmapsOfScreen, MinCmapsOfScreen, PlanesOfScreen, RootWindowOfScreen, WidthOfScreen, WidthMMOfScreen \- screen information functions and macros
 .SH SYNTAX
diff --git a/man/DisplayOfCCC.man b/man/DisplayOfCCC.man
index 4fc6153..512a173 100644
--- a/man/DisplayOfCCC.man
+++ b/man/DisplayOfCCC.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH DisplayOfCCC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH DisplayOfCCC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 DisplayOfCCC, VisualOfCCC, ScreenNumberOfCCC, ScreenWhitePointOfCCC, ClientWhitePointOfCCC \- Color Conversion Context macros
 .SH SYNTAX
diff --git a/man/ImageByteOrder.man b/man/ImageByteOrder.man
index c54d639..3782750 100644
--- a/man/ImageByteOrder.man
+++ b/man/ImageByteOrder.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH ImageByteOrder 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH ImageByteOrder __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 ImageByteOrder, BitmapBitOrder, BitmapPad, BitmapUnit, DisplayHeight, DisplayHeightMM, DisplayWidth, DisplayWidthMM, XListPixmapFormats, XPixmapFormatValues \- image format functions and macros
 .SH SYNTAX
diff --git a/man/IsCursorKey.man b/man/IsCursorKey.man
index cd33be5..d37ea1b 100644
--- a/man/IsCursorKey.man
+++ b/man/IsCursorKey.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH IsCursorKey 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH IsCursorKey __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 IsCursorKey, IsFunctionKey, IsKeypadKey, IsMiscFunctionKey, IsModifierKey, IsPFKey, IsPrivateKeypadKey \- keysym classification macros
 .SH SYNTAX
diff --git a/man/XAddConnectionWatch.man b/man/XAddConnectionWatch.man
index 8fb06dc..e1320c8 100644
--- a/man/XAddConnectionWatch.man
+++ b/man/XAddConnectionWatch.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XAddConnectionWatch 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAddConnectionWatch __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAddConnectionWatch, XRemoveConnectionWatch, XProcessInternalConnection, XInternalConnectionNumbers \- handle Xlib internal connections
 .SH SYNTAX
diff --git a/man/XAddHost.man b/man/XAddHost.man
index 01c3335..b9f0aa7 100644
--- a/man/XAddHost.man
+++ b/man/XAddHost.man
@@ -147,7 +147,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XAddHost 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAddHost __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAddHost, XAddHosts, XListHosts, XRemoveHost, XRemoveHosts, XSetAccessControl, XEnableAccessControl, XDisableAccessControl, XHostAddress, XServerInterpretedAddress \- control host access and host control structure
 .SH SYNTAX
diff --git a/man/XAllocClassHint.man b/man/XAllocClassHint.man
index 9699899..5145dd0 100644
--- a/man/XAllocClassHint.man
+++ b/man/XAllocClassHint.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XAllocClassHint 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAllocClassHint __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAllocClassHint, XSetClassHint, XGetClassHint, XClassHint \- allocate class hints structure and set or read a window's WM_CLASS property
 .SH SYNTAX
diff --git a/man/XAllocColor.man b/man/XAllocColor.man
index 57a3dd5..ef379b0 100644
--- a/man/XAllocColor.man
+++ b/man/XAllocColor.man
@@ -142,7 +142,7 @@
 ..
 .ny0
 '\" e
-.TH XAllocColor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAllocColor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAllocColor, XAllocNamedColor, XAllocColorCells, XAllocColorPlanes, XFreeColors \- allocate and free colors
 .SH SYNTAX
diff --git a/man/XAllocIconSize.man b/man/XAllocIconSize.man
index 466aa99..fb05d59 100644
--- a/man/XAllocIconSize.man
+++ b/man/XAllocIconSize.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XAllocIconSize 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAllocIconSize __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAllocIconSize, XSetIconSizes, XGetIconSizes, XIconSize \- allocate icon size structure and set or read a window's WM_ICON_SIZES property
 .SH SYNTAX
diff --git a/man/XAllocSizeHints.man b/man/XAllocSizeHints.man
index e086e44..e957214 100644
--- a/man/XAllocSizeHints.man
+++ b/man/XAllocSizeHints.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XAllocSizeHints 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAllocSizeHints __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAllocSizeHints, XSetWMNormalHints, XGetWMNormalHints, XSetWMSizeHints, XGetWMSizeHints, XSizeHints \- allocate size hints structure and set or read a window's WM_NORMAL_HINTS property
 .SH SYNTAX
diff --git a/man/XAllocStandardColormap.man b/man/XAllocStandardColormap.man
index 54a4a45..29039e5 100644
--- a/man/XAllocStandardColormap.man
+++ b/man/XAllocStandardColormap.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XAllocStandardColormap 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAllocStandardColormap __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAllocStandardColormap, XSetRGBColormaps, XGetRGBColormaps, XStandardColormap \- allocate, set, or read a standard colormap structure
 .SH SYNTAX
diff --git a/man/XAllocWMHints.man b/man/XAllocWMHints.man
index 7a3ce55..4c3fe1d 100644
--- a/man/XAllocWMHints.man
+++ b/man/XAllocWMHints.man
@@ -144,7 +144,7 @@
 ..
 .ny0
 '\" t
-.TH XAllocWMHints 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAllocWMHints __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAllocWMHints, XSetWMHints, XGetWMHints, XWMHints \- allocate window manager hints structure and set or read a window's WM_HINTS property
 .SH SYNTAX
diff --git a/man/XAllowEvents.man b/man/XAllowEvents.man
index 0805afa..8be5809 100644
--- a/man/XAllowEvents.man
+++ b/man/XAllowEvents.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XAllowEvents 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAllowEvents __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAllowEvents \- release queued events
 .SH SYNTAX
diff --git a/man/XAnyEvent.man b/man/XAnyEvent.man
index 5f6fc03..f095958 100644
--- a/man/XAnyEvent.man
+++ b/man/XAnyEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XAnyEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XAnyEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XAnyEvent, XEvent \- generic X event structures
 .SH STRUCTURES
diff --git a/man/XButtonEvent.man b/man/XButtonEvent.man
index f84e95b..4a28ea8 100644
--- a/man/XButtonEvent.man
+++ b/man/XButtonEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XButtonEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XButtonEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XButtonEvent, XKeyEvent, XMotionEvent \- KeyPress, KeyRelease, ButtonPress, ButtonRelease, and MotionNotify event structures
 .SH STRUCTURES
diff --git a/man/XChangeKeyboardControl.man b/man/XChangeKeyboardControl.man
index 841a7ed..58a5c9b 100644
--- a/man/XChangeKeyboardControl.man
+++ b/man/XChangeKeyboardControl.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XChangeKeyboardControl 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XChangeKeyboardControl __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XChangeKeyboardControl, XGetKeyboardControl, XAutoRepeatOn, XAutoRepeatOff, XBell, XQueryKeymap, XKeyboardControl \- manipulate keyboard settings and keyboard control structure
 .SH SYNTAX
diff --git a/man/XChangeKeyboardMapping.man b/man/XChangeKeyboardMapping.man
index 3d140cc..67b030a 100644
--- a/man/XChangeKeyboardMapping.man
+++ b/man/XChangeKeyboardMapping.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XChangeKeyboardMapping 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XChangeKeyboardMapping __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XChangeKeyboardMapping, XGetKeyboardMapping, XDisplayKeycodes, XSetModifierMapping, XGetModifierMapping, XNewModifiermap, XInsertModifiermapEntry, XDeleteModifiermapEntry, XFreeModifierMap XModifierKeymap \- manipulate keyboard encoding and keyboard encoding structure
 .SH SYNTAX
diff --git a/man/XChangePointerControl.man b/man/XChangePointerControl.man
index 4002543..2580432 100644
--- a/man/XChangePointerControl.man
+++ b/man/XChangePointerControl.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XChangePointerControl 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XChangePointerControl __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XChangePointerControl, XGetPointerControl \- control pointer 
 .SH SYNTAX
diff --git a/man/XChangeSaveSet.man b/man/XChangeSaveSet.man
index b699890..8ccc6c9 100644
--- a/man/XChangeSaveSet.man
+++ b/man/XChangeSaveSet.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XChangeSaveSet 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XChangeSaveSet __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XChangeSaveSet, XAddToSaveSet, XRemoveFromSaveSet \- change a client's save set
 .SH SYNTAX
diff --git a/man/XChangeWindowAttributes.man b/man/XChangeWindowAttributes.man
index 4e2a0af..1de7524 100644
--- a/man/XChangeWindowAttributes.man
+++ b/man/XChangeWindowAttributes.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XChangeWindowAttributes 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XChangeWindowAttributes __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XChangeWindowAttributes, XSetWindowBackground, XSetWindowBackgroundPixmap, XSetWindowBorder, XSetWindowBorderPixmap, XSetWindowColormap \- change window attributes
 .SH SYNTAX
diff --git a/man/XCirculateEvent.man b/man/XCirculateEvent.man
index e936dce..b3e4d8d 100644
--- a/man/XCirculateEvent.man
+++ b/man/XCirculateEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCirculateEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCirculateEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCirculateEvent \- CirculateNotify event structure
 .SH STRUCTURES
diff --git a/man/XCirculateRequestEvent.man b/man/XCirculateRequestEvent.man
index fdcd28b..ce97c26 100644
--- a/man/XCirculateRequestEvent.man
+++ b/man/XCirculateRequestEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCirculateRequestEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCirculateRequestEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCirculateRequestEvent \- CirculateRequest event structure
 .SH STRUCTURES
diff --git a/man/XClearArea.man b/man/XClearArea.man
index 590a8a6..26f2db2 100644
--- a/man/XClearArea.man
+++ b/man/XClearArea.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XClearArea 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XClearArea __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XClearArea, XClearWindow \- clear area or window
 .SH SYNTAX
diff --git a/man/XClientMessageEvent.man b/man/XClientMessageEvent.man
index 8fc4705..915cc55 100644
--- a/man/XClientMessageEvent.man
+++ b/man/XClientMessageEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XClientMessageEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XClientMessageEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XClientMessageEvent \- ClientMessage event structure
 .SH STRUCTURES
diff --git a/man/XColormapEvent.man b/man/XColormapEvent.man
index 28c65eb..95de9c9 100644
--- a/man/XColormapEvent.man
+++ b/man/XColormapEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XColormapEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XColormapEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XColormapEvent \- ColormapNotify event structure
 .SH STRUCTURES
diff --git a/man/XConfigureEvent.man b/man/XConfigureEvent.man
index b4124b0..5cc65cf 100644
--- a/man/XConfigureEvent.man
+++ b/man/XConfigureEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XConfigureEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XConfigureEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XConfigureEvent \- ConfigureNotify event structure
 .SH STRUCTURES
diff --git a/man/XConfigureRequestEvent.man b/man/XConfigureRequestEvent.man
index 4b78ed6..dbf6c99 100644
--- a/man/XConfigureRequestEvent.man
+++ b/man/XConfigureRequestEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XConfigureRequestEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XConfigureRequestEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XConfigureRequestEvent \- ConfigureRequest event structure
 .SH STRUCTURES
diff --git a/man/XConfigureWindow.man b/man/XConfigureWindow.man
index 4043cfa..54059c8 100644
--- a/man/XConfigureWindow.man
+++ b/man/XConfigureWindow.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XConfigureWindow 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XConfigureWindow __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XConfigureWindow, XMoveWindow, XResizeWindow, XMoveResizeWindow, XSetWindowBorderWidth, XWindowChanges \- configure windows and window changes structure
 .SH SYNTAX
diff --git a/man/XCopyArea.man b/man/XCopyArea.man
index 4ef78af..61f8ca2 100644
--- a/man/XCopyArea.man
+++ b/man/XCopyArea.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCopyArea 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCopyArea __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCopyArea, XCopyPlane \- copy areas
 .SH SYNTAX
diff --git a/man/XCreateColormap.man b/man/XCreateColormap.man
index ef3ce01..17c6dbf 100644
--- a/man/XCreateColormap.man
+++ b/man/XCreateColormap.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreateColormap 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateColormap __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateColormap, XCopyColormapAndFree, XFreeColormap, XColor \- create, copy, or destroy colormaps and color structure
 .SH SYNTAX
diff --git a/man/XCreateFontCursor.man b/man/XCreateFontCursor.man
index 79830f8..ce0cd5c 100644
--- a/man/XCreateFontCursor.man
+++ b/man/XCreateFontCursor.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreateFontCursor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateFontCursor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateFontCursor, XCreatePixmapCursor, XCreateGlyphCursor \- create cursors
 .SH SYNTAX
diff --git a/man/XCreateFontSet.man b/man/XCreateFontSet.man
index 023aed4..3cfe8a1 100644
--- a/man/XCreateFontSet.man
+++ b/man/XCreateFontSet.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreateFontSet 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateFontSet __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateFontSet, XFreeFontSet \- create and free an international text drawing font set
 .SH SYNTAX
diff --git a/man/XCreateGC.man b/man/XCreateGC.man
index a48839f..91897a0 100644
--- a/man/XCreateGC.man
+++ b/man/XCreateGC.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XCreateGC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateGC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateGC, XCopyGC, XChangeGC, XGetGCValues, XFreeGC, XGContextFromGC, XGCValues \- create or free graphics contexts and graphics context structure
 .SH SYNTAX
diff --git a/man/XCreateIC.man b/man/XCreateIC.man
index f4b7c7c..9a1901e 100644
--- a/man/XCreateIC.man
+++ b/man/XCreateIC.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreateIC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateIC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateIC, XDestroyIC, XIMOfIC \- create, destroy, and obtain the input method of an input context
 .SH SYNTAX
diff --git a/man/XCreateOC.man b/man/XCreateOC.man
index 5e4a6d8..7f7eb0c 100644
--- a/man/XCreateOC.man
+++ b/man/XCreateOC.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreateOC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateOC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateOC, XDestroyOC, XSetOCValues, XGetOCValues, XOMOfOC \- create output contexts
 .SH SYNTAX
diff --git a/man/XCreatePixmap.man b/man/XCreatePixmap.man
index d4bdd93..15938ca 100644
--- a/man/XCreatePixmap.man
+++ b/man/XCreatePixmap.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreatePixmap 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreatePixmap __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreatePixmap, XFreePixmap \- create or destroy pixmaps
 .SH SYNTAX
diff --git a/man/XCreateRegion.man b/man/XCreateRegion.man
index d43cf2a..808d7a1 100644
--- a/man/XCreateRegion.man
+++ b/man/XCreateRegion.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreateRegion 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateRegion __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateRegion, XSetRegion, XDestroyRegion \- create or destroy regions
 .SH SYNTAX
diff --git a/man/XCreateWindow.man b/man/XCreateWindow.man
index df0102b..18c66df 100644
--- a/man/XCreateWindow.man
+++ b/man/XCreateWindow.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XCreateWindow 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateWindow __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateWindow, XCreateSimpleWindow, XSetWindowAttributes \- create windows and window attributes structure
 .SH SYNTAX
diff --git a/man/XCreateWindowEvent.man b/man/XCreateWindowEvent.man
index fc950d8..9fb2ce2 100644
--- a/man/XCreateWindowEvent.man
+++ b/man/XCreateWindowEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreateWindowEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateWindowEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCreateWindowEvent \- CreateNotify event structure
 .SH STRUCTURES
diff --git a/man/XCrossingEvent.man b/man/XCrossingEvent.man
index 2cc1559..9721a2c 100644
--- a/man/XCrossingEvent.man
+++ b/man/XCrossingEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCrossingEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCrossingEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XCrossingEvent \- EnterNotify and LeaveNotify event structure
 .SH STRUCTURES
diff --git a/man/XDefineCursor.man b/man/XDefineCursor.man
index 46fa149..a3000d2 100644
--- a/man/XDefineCursor.man
+++ b/man/XDefineCursor.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDefineCursor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDefineCursor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDefineCursor, XUndefineCursor \- define cursors
 .SH SYNTAX
diff --git a/man/XDestroyWindow.man b/man/XDestroyWindow.man
index cdd4736..ca44134 100644
--- a/man/XDestroyWindow.man
+++ b/man/XDestroyWindow.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDestroyWindow 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDestroyWindow __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDestroyWindow, XDestroySubwindows \- destroy windows
 .SH SYNTAX
diff --git a/man/XDestroyWindowEvent.man b/man/XDestroyWindowEvent.man
index 2a1b0cd..9c5a261 100644
--- a/man/XDestroyWindowEvent.man
+++ b/man/XDestroyWindowEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDestroyWindowEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDestroyWindowEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDestroyWindowEvent \- DestroyNotify event structure
 .SH STRUCTURES
diff --git a/man/XDrawArc.man b/man/XDrawArc.man
index 73609f3..c878c45 100644
--- a/man/XDrawArc.man
+++ b/man/XDrawArc.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" e
-.TH XDrawArc 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDrawArc __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDrawArc, XDrawArcs, XArc \- draw arcs and arc structure
 .SH SYNTAX
diff --git a/man/XDrawImageString.man b/man/XDrawImageString.man
index 1e13ee1..38a5811 100644
--- a/man/XDrawImageString.man
+++ b/man/XDrawImageString.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDrawImageString 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDrawImageString __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDrawImageString, XDrawImageString16 \- draw image text
 .SH SYNTAX
diff --git a/man/XDrawLine.man b/man/XDrawLine.man
index 7beba73..03ae345 100644
--- a/man/XDrawLine.man
+++ b/man/XDrawLine.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDrawLine 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDrawLine __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDrawLine, XDrawLines, XDrawSegments, XSegment \- draw lines, polygons, and line structure
 .SH SYNTAX
diff --git a/man/XDrawPoint.man b/man/XDrawPoint.man
index dff55a5..37be5f5 100644
--- a/man/XDrawPoint.man
+++ b/man/XDrawPoint.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDrawPoint 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDrawPoint __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDrawPoint, XDrawPoints, XPoint \- draw points and points structure
 .SH SYNTAX
diff --git a/man/XDrawRectangle.man b/man/XDrawRectangle.man
index 5a36f82..a210cad 100644
--- a/man/XDrawRectangle.man
+++ b/man/XDrawRectangle.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDrawRectangle 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDrawRectangle __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDrawRectangle, XDrawRectangles, XRectangle \- draw rectangles and rectangles structure
 .SH SYNTAX
diff --git a/man/XDrawString.man b/man/XDrawString.man
index 6e8e8b5..767be99 100644
--- a/man/XDrawString.man
+++ b/man/XDrawString.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDrawString 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDrawString __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDrawString, XDrawString16 \- draw text characters
 .SH SYNTAX
diff --git a/man/XDrawText.man b/man/XDrawText.man
index fd7a266..706d69e 100644
--- a/man/XDrawText.man
+++ b/man/XDrawText.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XDrawText 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XDrawText __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XDrawText, XDrawText16, XTextItem, XTextItem16 \- draw polytext text and text drawing structures
 .SH SYNTAX
diff --git a/man/XEmptyRegion.man b/man/XEmptyRegion.man
index 723f4f9..3721199 100644
--- a/man/XEmptyRegion.man
+++ b/man/XEmptyRegion.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XEmptyRegion 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XEmptyRegion __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XEmptyRegion, XEqualRegion, XPointInRegion, XRectInRegion \- determine if regions are empty or equal
 .SH SYNTAX
diff --git a/man/XErrorEvent.man b/man/XErrorEvent.man
index b4d7dfa..e1f4eae 100644
--- a/man/XErrorEvent.man
+++ b/man/XErrorEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XErrorEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XErrorEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XErrorEvent \- X error event structure
 .SH STRUCTURES
diff --git a/man/XExposeEvent.man b/man/XExposeEvent.man
index 18da905..82efac2 100644
--- a/man/XExposeEvent.man
+++ b/man/XExposeEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XExposeEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XExposeEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XExposeEvent \- Expose event structure
 .SH STRUCTURES
diff --git a/man/XExtentsOfFontSet.man b/man/XExtentsOfFontSet.man
index ed5fbff..35d4c7b 100644
--- a/man/XExtentsOfFontSet.man
+++ b/man/XExtentsOfFontSet.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XExtentsOfFontSet 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XExtentsOfFontSet __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XExtentsOfFontSet \- obtain the maximum extents structure for a font set
 .SH SYNTAX
diff --git a/man/XFillRectangle.man b/man/XFillRectangle.man
index f12d76a..5061f72 100644
--- a/man/XFillRectangle.man
+++ b/man/XFillRectangle.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XFillRectangle 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XFillRectangle __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XFillRectangle, XFillRectangles, XFillPolygon, XFillArc, XFillArcs \- fill rectangles, polygons, or arcs
 .SH SYNTAX
diff --git a/man/XFilterEvent.man b/man/XFilterEvent.man
index bda0a0d..ea6bdce 100644
--- a/man/XFilterEvent.man
+++ b/man/XFilterEvent.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XFilterEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XFilterEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XFilterEvent \- filter X events for an input method
 .SH SYNTAX
diff --git a/man/XFlush.man b/man/XFlush.man
index dd2b9b4..fa3fc36 100644
--- a/man/XFlush.man
+++ b/man/XFlush.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XFlush 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XFlush __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XFlush, XSync, XEventsQueued, XPending \- handle output buffer or event queue
 .SH SYNTAX
diff --git a/man/XFocusChangeEvent.man b/man/XFocusChangeEvent.man
index dd0955b..c96d3cc 100644
--- a/man/XFocusChangeEvent.man
+++ b/man/XFocusChangeEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XFocusChangeEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XFocusChangeEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XFocusChangeEvent \- FocusIn and FocusOut event structure
 .SH STRUCTURES
diff --git a/man/XFontSetExtents.man b/man/XFontSetExtents.man
index da8ebd5..6d415ba 100644
--- a/man/XFontSetExtents.man
+++ b/man/XFontSetExtents.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XFontSetExtents 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XFontSetExtents __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XFontSetExtents \- XFontSetExtents structure
 .SH STRUCTURES
diff --git a/man/XFontsOfFontSet.man b/man/XFontsOfFontSet.man
index 360c52c..7e2dc47 100644
--- a/man/XFontsOfFontSet.man
+++ b/man/XFontsOfFontSet.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XFontsOfFontSet 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XFontsOfFontSet __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XFontsOfFontSet, XBaseFontNameListOfFontSet, XLocaleOfFontSet, XContextDependentDrawing, XContextualDrawing, XDirectionalDependentDrawing \- obtain fontset information
 .SH SYNTAX
diff --git a/man/XFree.man b/man/XFree.man
index 34c8c8b..5952998 100644
--- a/man/XFree.man
+++ b/man/XFree.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XFree 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XFree __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XFree \- free client data
 .SH SYNTAX
diff --git a/man/XGetVisualInfo.man b/man/XGetVisualInfo.man
index 5df1f8f..86f6856 100644
--- a/man/XGetVisualInfo.man
+++ b/man/XGetVisualInfo.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XGetVisualInfo 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGetVisualInfo __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGetVisualInfo, XMatchVisualInfo, XVisualIDFromVisual, XVisualInfo \- obtain visual information and visual structure
 .SH SYNTAX
diff --git a/man/XGetWindowAttributes.man b/man/XGetWindowAttributes.man
index a2c5eaf..d0521ad 100644
--- a/man/XGetWindowAttributes.man
+++ b/man/XGetWindowAttributes.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XGetWindowAttributes 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGetWindowAttributes __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGetWindowAttributes, XGetGeometry, XWindowAttributes \- get current window attribute or geometry and current window attributes structure
 .SH SYNTAX
diff --git a/man/XGetWindowProperty.man b/man/XGetWindowProperty.man
index 8db9f7c..a1d19ff 100644
--- a/man/XGetWindowProperty.man
+++ b/man/XGetWindowProperty.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XGetWindowProperty 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGetWindowProperty __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGetWindowProperty, XListProperties, XChangeProperty, XRotateWindowProperties, XDeleteProperty \- obtain and change window properties
 .SH SYNTAX
diff --git a/man/XGrabButton.man b/man/XGrabButton.man
index 03db7c8..a6c20fe 100644
--- a/man/XGrabButton.man
+++ b/man/XGrabButton.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XGrabButton 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGrabButton __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGrabButton, XUngrabButton \- grab pointer buttons
 .SH SYNTAX
diff --git a/man/XGrabKey.man b/man/XGrabKey.man
index bfe9270..b0cfd8b 100644
--- a/man/XGrabKey.man
+++ b/man/XGrabKey.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XGrabKey 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGrabKey __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGrabKey, XUngrabKey \- grab keyboard keys
 .SH SYNTAX
diff --git a/man/XGrabKeyboard.man b/man/XGrabKeyboard.man
index 340e11e..db1255f 100644
--- a/man/XGrabKeyboard.man
+++ b/man/XGrabKeyboard.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XGrabKeyboard 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGrabKeyboard __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGrabKeyboard, XUngrabKeyboard \- grab the keyboard
 .SH SYNTAX
diff --git a/man/XGrabPointer.man b/man/XGrabPointer.man
index c4756a1..46a9986 100644
--- a/man/XGrabPointer.man
+++ b/man/XGrabPointer.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XGrabPointer 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGrabPointer __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGrabPointer, XUngrabPointer, XChangeActivePointerGrab \- grab the pointer
 .SH SYNTAX
diff --git a/man/XGrabServer.man b/man/XGrabServer.man
index 7f318bd..e3701c7 100644
--- a/man/XGrabServer.man
+++ b/man/XGrabServer.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XGrabServer 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGrabServer __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGrabServer, XUngrabServer \- grab the server
 .SH SYNTAX
diff --git a/man/XGraphicsExposeEvent.man b/man/XGraphicsExposeEvent.man
index 7e481c9..6e20083 100644
--- a/man/XGraphicsExposeEvent.man
+++ b/man/XGraphicsExposeEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XGraphicsExposeEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGraphicsExposeEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGraphicsExposeEvent, XNoExposeEvent \- GraphicsExpose and NoExpose event structures
 .SH STRUCTURES
diff --git a/man/XGravityEvent.man b/man/XGravityEvent.man
index fd825df..9d07e6f 100644
--- a/man/XGravityEvent.man
+++ b/man/XGravityEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XGravityEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XGravityEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XGravityEvent \- GravityNotify event structure
 .SH STRUCTURES
diff --git a/man/XIconifyWindow.man b/man/XIconifyWindow.man
index a863c13..edb1101 100644
--- a/man/XIconifyWindow.man
+++ b/man/XIconifyWindow.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XIconifyWindow 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XIconifyWindow __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XIconifyWindow, XWithdrawWindow, XReconfigureWMWindow \- manipulate top-level windows
 .SH SYNTAX
diff --git a/man/XIfEvent.man b/man/XIfEvent.man
index 90dad9e..61d28e5 100644
--- a/man/XIfEvent.man
+++ b/man/XIfEvent.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XIfEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XIfEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XIfEvent, XCheckIfEvent, XPeekIfEvent \- check the event queue with a predicate procedure
 .SH SYNTAX
diff --git a/man/XInitImage.man b/man/XInitImage.man
index 3de1b41..e2c82a6 100644
--- a/man/XInitImage.man
+++ b/man/XInitImage.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XCreateImage 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XCreateImage __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XInitImage, XCreateImage, XGetPixel, XPutPixel, XSubImage, XAddPixel, XDestroyImage \- image utilities
 .SH SYNTAX
diff --git a/man/XInitThreads.man b/man/XInitThreads.man
index 92edbe6..64b1384 100644
--- a/man/XInitThreads.man
+++ b/man/XInitThreads.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XInitThreads 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XInitThreads __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XInitThreads, XLockDisplay, XUnlockDisplay \- multi-threading support
 .SH SYNTAX
diff --git a/man/XInstallColormap.man b/man/XInstallColormap.man
index 0650ab2..249ba49 100644
--- a/man/XInstallColormap.man
+++ b/man/XInstallColormap.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XInstallColormap 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XInstallColormap __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XInstallColormap, XUninstallColormap, XListInstalledColormaps \- control colormaps
 .SH SYNTAX
diff --git a/man/XInternAtom.man b/man/XInternAtom.man
index b0c103e..4c7b56f 100644
--- a/man/XInternAtom.man
+++ b/man/XInternAtom.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XInternAtom 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XInternAtom __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XInternAtom, XInternAtoms, XGetAtomName, XGetAtomNames \- create or return atom names
 .SH SYNTAX
diff --git a/man/XIntersectRegion.man b/man/XIntersectRegion.man
index 0945033..c4482f5 100644
--- a/man/XIntersectRegion.man
+++ b/man/XIntersectRegion.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XIntersectRegion 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XIntersectRegion __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XIntersectRegion, XUnionRegion, XUnionRectWithRegion, XSubtractRegion, XXorRegion, XOffsetRegion, XShrinkRegion \- region arithmetic
 .SH SYNTAX
diff --git a/man/XKeymapEvent.man b/man/XKeymapEvent.man
index 2b833ae..c687c96 100644
--- a/man/XKeymapEvent.man
+++ b/man/XKeymapEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XKeymapEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XKeymapEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XKeymapEvent \- KeymapNotify event structure
 .SH STRUCTURES
diff --git a/man/XListFonts.man b/man/XListFonts.man
index 726bced..41a88ee 100644
--- a/man/XListFonts.man
+++ b/man/XListFonts.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XListFonts 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XListFonts __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XListFonts, XFreeFontNames, XListFontsWithInfo, XFreeFontInfo \- obtain or free font names and information
 .SH SYNTAX
diff --git a/man/XLoadFont.man b/man/XLoadFont.man
index cb7a387..ab7e890 100644
--- a/man/XLoadFont.man
+++ b/man/XLoadFont.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XLoadFont 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XLoadFont __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XLoadFont, XQueryFont, XLoadQueryFont, XFreeFont, XGetFontProperty, XUnloadFont, XCharStruct, XFontProp, XChar2b, XFontStruct \- load or unload fonts and font metric structures
 .SH SYNTAX
diff --git a/man/XLookupKeysym.man b/man/XLookupKeysym.man
index 38a5900..7e1ac4a 100644
--- a/man/XLookupKeysym.man
+++ b/man/XLookupKeysym.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XLookupKeysym 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XLookupKeysym __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XLookupKeysym, XRefreshKeyboardMapping, XLookupString, XRebindKeysym \- handle keyboard input events in Latin-1
 .SH SYNTAX
diff --git a/man/XMapEvent.man b/man/XMapEvent.man
index 097b310..5bc8f85 100644
--- a/man/XMapEvent.man
+++ b/man/XMapEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XMapEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XMapEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XMapEvent, XMappingEvent \- MapNotify and MappingNotify event structures
 .SH STRUCTURES
diff --git a/man/XMapRequestEvent.man b/man/XMapRequestEvent.man
index 15ae361..f76551d 100644
--- a/man/XMapRequestEvent.man
+++ b/man/XMapRequestEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XMapRequestEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XMapRequestEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XMapRequestEvent \- MapRequest event structure
 .SH STRUCTURES
diff --git a/man/XMapWindow.man b/man/XMapWindow.man
index 5dfb4ce..b56fa82 100644
--- a/man/XMapWindow.man
+++ b/man/XMapWindow.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XMapWindow 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XMapWindow __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XMapWindow, XMapRaised, XMapSubwindows \- map windows
 .SH SYNTAX
diff --git a/man/XNextEvent.man b/man/XNextEvent.man
index cfed368..cef7485 100644
--- a/man/XNextEvent.man
+++ b/man/XNextEvent.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XNextEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XNextEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XNextEvent, XPeekEvent, XWindowEvent, XCheckWindowEvent, XMaskEvent, XCheckMaskEvent, XCheckTypedEvent, XCheckTypedWindowEvent \- select events by type
 .SH SYNTAX
diff --git a/man/XNoOp.man b/man/XNoOp.man
index 10ce6a2..9fd9890 100644
--- a/man/XNoOp.man
+++ b/man/XNoOp.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XNoOp 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XNoOp __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XNoOp \- No Operation
 .SH SYNTAX
diff --git a/man/XOpenDisplay.man b/man/XOpenDisplay.man
index ddc26ee..04ae851 100644
--- a/man/XOpenDisplay.man
+++ b/man/XOpenDisplay.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XOpenDisplay 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XOpenDisplay __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XOpenDisplay, XCloseDisplay \- connect or disconnect to X server
 .SH SYNTAX
diff --git a/man/XOpenIM.man b/man/XOpenIM.man
index b2f8973..0470085 100644
--- a/man/XOpenIM.man
+++ b/man/XOpenIM.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XOpenIM 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XOpenIM __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XOpenIM, XCloseIM, XSetIMValues, XGetIMValues, XDisplayOfIM, XLocaleOfIM, XRegisterIMInstantiateCallback, XUnregisterIMInstantiateCallback \- open, close, and otain input method information
 .SH SYNTAX
diff --git a/man/XOpenOM.man b/man/XOpenOM.man
index 3977c55..498c016 100644
--- a/man/XOpenOM.man
+++ b/man/XOpenOM.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XOpenOM 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XOpenOM __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XOpenOM, XCloseOM, XSetOMValues, XGetOMValues, XDisplayOfOM, XLocaleOfOM \- open output methods
 .SH SYNTAX
diff --git a/man/XParseGeometry.man b/man/XParseGeometry.man
index a3b0a92..1f5068c 100644
--- a/man/XParseGeometry.man
+++ b/man/XParseGeometry.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XParseGeometry 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XParseGeometry __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XParseGeometry, XWMGeometry \- parse window geometry
 .SH SYNTAX
diff --git a/man/XPolygonRegion.man b/man/XPolygonRegion.man
index 97d5eb3..8be420c 100644
--- a/man/XPolygonRegion.man
+++ b/man/XPolygonRegion.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XPolygonRegion 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XPolygonRegion __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XPolygonRegion, XClipBox \- generate regions
 .SH SYNTAX
diff --git a/man/XPropertyEvent.man b/man/XPropertyEvent.man
index a9378ef..ef34642 100644
--- a/man/XPropertyEvent.man
+++ b/man/XPropertyEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XPropertyEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XPropertyEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XPropertyEvent \- PropertyNotify event structure
 .SH STRUCTURES
diff --git a/man/XPutBackEvent.man b/man/XPutBackEvent.man
index 6ab7ce2..0fc4bca 100644
--- a/man/XPutBackEvent.man
+++ b/man/XPutBackEvent.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XPutBackEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XPutBackEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XPutBackEvent \- put events back on the queue
 .SH SYNTAX
diff --git a/man/XPutImage.man b/man/XPutImage.man
index 10ccb4a..8880d8d 100644
--- a/man/XPutImage.man
+++ b/man/XPutImage.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XPutImage 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XPutImage __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XPutImage, XGetImage, XGetSubImage \- transfer images
 .SH SYNTAX
diff --git a/man/XQueryBestSize.man b/man/XQueryBestSize.man
index 65b70c6..667d15b 100644
--- a/man/XQueryBestSize.man
+++ b/man/XQueryBestSize.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XQueryBestSize 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XQueryBestSize __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XQueryBestSize, XQueryBestTile, XQueryBestStipple \- determine efficient sizes
 .SH SYNTAX
diff --git a/man/XQueryColor.man b/man/XQueryColor.man
index 4506286..eff8a6c 100644
--- a/man/XQueryColor.man
+++ b/man/XQueryColor.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XQueryColor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XQueryColor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XQueryColor, XQueryColors, XLookupColor, XParseColor \- obtain color values
 .SH SYNTAX
diff --git a/man/XQueryExtension.man b/man/XQueryExtension.man
index cb74f06..a81ec0c 100644
--- a/man/XQueryExtension.man
+++ b/man/XQueryExtension.man
@@ -15,7 +15,7 @@
 .\" express or implied warranty.
 .\"
 .dS xL Programming with Xlib
-.TH XQueryExtension 3X11 7.0 Release "X Version 11" "X FUNCTIONS"
+.TH XQueryExtension __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XQueryExtension, XListExtensions, XFreeExtensionList \- list available extensions
 .SH SYNTAX
diff --git a/man/XQueryPointer.man b/man/XQueryPointer.man
index 440d43b..708ef2d 100644
--- a/man/XQueryPointer.man
+++ b/man/XQueryPointer.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XQueryPointer 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XQueryPointer __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XQueryPointer \- get pointer coordinates
 .SH SYNTAX
diff --git a/man/XQueryTree.man b/man/XQueryTree.man
index a1619d8..81338d2 100644
--- a/man/XQueryTree.man
+++ b/man/XQueryTree.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XQueryTree 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XQueryTree __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XQueryTree \- query window tree information
 .SH SYNTAX
diff --git a/man/XRaiseWindow.man b/man/XRaiseWindow.man
index af66c6a..5acb18d 100644
--- a/man/XRaiseWindow.man
+++ b/man/XRaiseWindow.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XRaiseWindow 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XRaiseWindow __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XRaiseWindow, XLowerWindow, XCirculateSubwindows, XCirculateSubwindowsUp, XCirculateSubwindowsDown, XRestackWindows \- change window stacking order
 .SH SYNTAX
diff --git a/man/XReadBitmapFile.man b/man/XReadBitmapFile.man
index 1880f37..defbbf9 100644
--- a/man/XReadBitmapFile.man
+++ b/man/XReadBitmapFile.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XReadBitmapFile 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XReadBitmapFile __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XReadBitmapFile, XReadBitmapFileData, XWriteBitmapFile, XCreatePixmapFromBitmapData, XCreateBitmapFromData \- manipulate bitmaps
 .SH SYNTAX
diff --git a/man/XRecolorCursor.man b/man/XRecolorCursor.man
index 4b005ca..07c036a 100644
--- a/man/XRecolorCursor.man
+++ b/man/XRecolorCursor.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XRecolorCursor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XRecolorCursor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XRecolorCursor, XFreeCursor, XQueryBestCursor \- manipulate cursors
 .SH SYNTAX
diff --git a/man/XReparentEvent.man b/man/XReparentEvent.man
index f5927fa..ff42e28 100644
--- a/man/XReparentEvent.man
+++ b/man/XReparentEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XReparentEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XReparentEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XReparentEvent \- ReparentNotify event structure
 .SH STRUCTURES
diff --git a/man/XReparentWindow.man b/man/XReparentWindow.man
index da1c1a8..ae3363f 100644
--- a/man/XReparentWindow.man
+++ b/man/XReparentWindow.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XReparentWindow 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XReparentWindow __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XReparentWindow \- reparent windows
 .SH SYNTAX
diff --git a/man/XResizeRequestEvent.man b/man/XResizeRequestEvent.man
index ee7959c..71243b9 100644
--- a/man/XResizeRequestEvent.man
+++ b/man/XResizeRequestEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XResizeRequestEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XResizeRequestEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XResizeRequestEvent \- ResizeRequest event structure
 .SH STRUCTURES
diff --git a/man/XResourceManagerString.man b/man/XResourceManagerString.man
index cc713a7..81f69b7 100644
--- a/man/XResourceManagerString.man
+++ b/man/XResourceManagerString.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XResourceManagerString 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XResourceManagerString __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XResourceManagerString, XScreenResourceString \- obtain server resource properties
 .SH SYNTAX
diff --git a/man/XSaveContext.man b/man/XSaveContext.man
index b76e561..566e251 100644
--- a/man/XSaveContext.man
+++ b/man/XSaveContext.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSaveContext 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSaveContext __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSaveContext, XFindContext, XDeleteContext, XUniqueContext \- associative look-up routines
 .SH SYNTAX
diff --git a/man/XSelectInput.man b/man/XSelectInput.man
index 1e3be47..5868748 100644
--- a/man/XSelectInput.man
+++ b/man/XSelectInput.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSelectInput 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSelectInput __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSelectInput \- select input events
 .SH SYNTAX
diff --git a/man/XSelectionClearEvent.man b/man/XSelectionClearEvent.man
index 3c5bacb..3fd7b1f 100644
--- a/man/XSelectionClearEvent.man
+++ b/man/XSelectionClearEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSelectionClearEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSelectionClearEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSelectionClearEvent \- SelectionClear event structure
 .SH STRUCTURES
diff --git a/man/XSelectionEvent.man b/man/XSelectionEvent.man
index 783d033..45481f5 100644
--- a/man/XSelectionEvent.man
+++ b/man/XSelectionEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSelectionEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSelectionEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSelectionEvent \- SelectionNotify event structure
 .SH STRUCTURES
diff --git a/man/XSelectionRequestEvent.man b/man/XSelectionRequestEvent.man
index 165a9e3..9d2febe 100644
--- a/man/XSelectionRequestEvent.man
+++ b/man/XSelectionRequestEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSelectionRequestEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSelectionRequestEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSelectionRequestEvent \- SelectionRequest event structure
 .SH STRUCTURES
diff --git a/man/XSendEvent.man b/man/XSendEvent.man
index 98f5ce5..5c89715 100644
--- a/man/XSendEvent.man
+++ b/man/XSendEvent.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSendEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSendEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSendEvent, XDisplayMotionBufferSize, XGetMotionEvents, XTimeCoord \- send events and pointer motion history structure
 .SH SYNTAX
diff --git a/man/XSetArcMode.man b/man/XSetArcMode.man
index f7bb944..0862e95 100644
--- a/man/XSetArcMode.man
+++ b/man/XSetArcMode.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetArcMode 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetArcMode __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetArcMode, XSetSubwindowMode, XSetGraphicsExposure \- GC convenience routines
 .SH SYNTAX
diff --git a/man/XSetClipOrigin.man b/man/XSetClipOrigin.man
index 498af34..ddbe6cf 100644
--- a/man/XSetClipOrigin.man
+++ b/man/XSetClipOrigin.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetClipOrigin 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetClipOrigin __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetClipOrigin, XSetClipMask, XSetClipRectangles \- GC convenience routines
 .SH SYNTAX
diff --git a/man/XSetCloseDownMode.man b/man/XSetCloseDownMode.man
index 9c3ed00..e16f80c 100644
--- a/man/XSetCloseDownMode.man
+++ b/man/XSetCloseDownMode.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetCloseDownMode 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetCloseDownMode __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetCloseDownMode, XKillClient \- control clients
 .SH SYNTAX
diff --git a/man/XSetCommand.man b/man/XSetCommand.man
index 3d86d30..c5f03e7 100644
--- a/man/XSetCommand.man
+++ b/man/XSetCommand.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetCommand 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetCommand __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetCommand, XGetCommand \- set or read a window's WM_COMMAND property
 .SH SYNTAX
diff --git a/man/XSetErrorHandler.man b/man/XSetErrorHandler.man
index c3acf27..5ccf3a9 100644
--- a/man/XSetErrorHandler.man
+++ b/man/XSetErrorHandler.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetErrorHandler 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetErrorHandler __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetErrorHandler, XGetErrorText, XDisplayName, XSetIOErrorHandler, XGetErrorDatabaseText \- default error handlers
 .SH SYNTAX
diff --git a/man/XSetFillStyle.man b/man/XSetFillStyle.man
index 85c56bf..e8700db 100644
--- a/man/XSetFillStyle.man
+++ b/man/XSetFillStyle.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetFillStyle 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetFillStyle __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetFillStyle, XSetFillRule \- GC convenience routines
 .SH SYNTAX
diff --git a/man/XSetFont.man b/man/XSetFont.man
index 7401c05..4b44e22 100644
--- a/man/XSetFont.man
+++ b/man/XSetFont.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetFont 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetFont __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetFont \- GC convenience routines
 .SH SYNTAX
diff --git a/man/XSetFontPath.man b/man/XSetFontPath.man
index bd91709..2548a5f 100644
--- a/man/XSetFontPath.man
+++ b/man/XSetFontPath.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetFontPath 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetFontPath __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetFontPath, XGetFontPath, XFreeFontPath \- set, get, or free the font search path
 .SH SYNTAX
diff --git a/man/XSetICFocus.man b/man/XSetICFocus.man
index 1e6dd6e..19b8184 100644
--- a/man/XSetICFocus.man
+++ b/man/XSetICFocus.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetICFocus 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetICFocus __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetICFocus, XUnsetICFocus \- set and unset input context focus
 .SH SYNTAX
diff --git a/man/XSetICValues.man b/man/XSetICValues.man
index c7aca29..1e072f7 100644
--- a/man/XSetICValues.man
+++ b/man/XSetICValues.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetICValues 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetICValues __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetICValues, XGetICValues \- set and obtain XIC values
 .SH SYNTAX
diff --git a/man/XSetInputFocus.man b/man/XSetInputFocus.man
index 821241b..14d2655 100644
--- a/man/XSetInputFocus.man
+++ b/man/XSetInputFocus.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetInputFocus 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetInputFocus __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetInputFocus, XGetInputFocus \- control input focus
 .SH SYNTAX
diff --git a/man/XSetLineAttributes.man b/man/XSetLineAttributes.man
index 7a903c4..09be3f5 100644
--- a/man/XSetLineAttributes.man
+++ b/man/XSetLineAttributes.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetLineAttributes 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetLineAttributes __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetLineAttributes, XSetDashes \- GC convenience routines
 .SH SYNTAX
diff --git a/man/XSetPointerMapping.man b/man/XSetPointerMapping.man
index da09bac..7b0845a 100644
--- a/man/XSetPointerMapping.man
+++ b/man/XSetPointerMapping.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetPointerMapping 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetPointerMapping __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetPointerMapping, XGetPointerMapping \- manipulate pointer settings
 .SH SYNTAX
diff --git a/man/XSetScreenSaver.man b/man/XSetScreenSaver.man
index 0632ae5..c0c1e36 100644
--- a/man/XSetScreenSaver.man
+++ b/man/XSetScreenSaver.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetScreenSaver 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetScreenSaver __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetScreenSaver, XForceScreenSaver, XActivateScreenSaver, XResetScreenSaver, XGetScreenSaver \- manipulate the screen saver
 .SH SYNTAX
diff --git a/man/XSetSelectionOwner.man b/man/XSetSelectionOwner.man
index 7ff67ed..64ad976 100644
--- a/man/XSetSelectionOwner.man
+++ b/man/XSetSelectionOwner.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetSelectionOwner 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetSelectionOwner __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetSelectionOwner, XGetSelectionOwner, XConvertSelection \- manipulate window selection
 .SH SYNTAX
diff --git a/man/XSetState.man b/man/XSetState.man
index 2148567..aa0143d 100644
--- a/man/XSetState.man
+++ b/man/XSetState.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetState 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetState __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetState, XSetFunction, XSetPlaneMask, XSetForeground, XSetBackground \- GC convenience routines
 .SH SYNTAX
diff --git a/man/XSetTextProperty.man b/man/XSetTextProperty.man
index 24b6fc9..b7ae9ea 100644
--- a/man/XSetTextProperty.man
+++ b/man/XSetTextProperty.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetTextProperty 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetTextProperty __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetTextProperty, XGetTextProperty \- set and read text properties
 .SH SYNTAX
diff --git a/man/XSetTile.man b/man/XSetTile.man
index f9e439d..0fa1e04 100644
--- a/man/XSetTile.man
+++ b/man/XSetTile.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetTile 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetTile __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetTile, XSetStipple, XSetTSOrigin \- GC convenience routines
 .SH SYNTAX
diff --git a/man/XSetTransientForHint.man b/man/XSetTransientForHint.man
index 4ace855..03c292d 100644
--- a/man/XSetTransientForHint.man
+++ b/man/XSetTransientForHint.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetTransientForHint 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetTransientForHint __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetTransientForHint, XGetTransientForHint \- set or read a window's WM_TRANSIENT_FOR property
 .SH SYNTAX
diff --git a/man/XSetWMClientMachine.man b/man/XSetWMClientMachine.man
index 8299ee1..504a594 100644
--- a/man/XSetWMClientMachine.man
+++ b/man/XSetWMClientMachine.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetWMClientMachine 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetWMClientMachine __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetWMClientMachine, XGetWMClientMachine \- set or read a window's WM_CLIENT_MACHINE property
 .SH SYNTAX
diff --git a/man/XSetWMColormapWindows.man b/man/XSetWMColormapWindows.man
index 015ec82..b9eca2e 100644
--- a/man/XSetWMColormapWindows.man
+++ b/man/XSetWMColormapWindows.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetWMColormapWindows 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetWMColormapWindows __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetWMColormapWindows, XGetWMColormapWindows \- set or read a window's WM_COLORMAP_WINDOWS property
 .SH SYNTAX
diff --git a/man/XSetWMIconName.man b/man/XSetWMIconName.man
index baef884..b389a90 100644
--- a/man/XSetWMIconName.man
+++ b/man/XSetWMIconName.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetWMIconName 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetWMIconName __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetWMIconName, XGetWMIconName, XSetIconName, XGetIconName \- set or read a window's WM_ICON_NAME property
 .SH SYNTAX
diff --git a/man/XSetWMName.man b/man/XSetWMName.man
index 0e963aa..5a04071 100644
--- a/man/XSetWMName.man
+++ b/man/XSetWMName.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetWMName 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetWMName __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetWMName, XGetWMName, XStoreName, XFetchName \- set or read a window's WM_NAME property
 .SH SYNTAX
diff --git a/man/XSetWMProperties.man b/man/XSetWMProperties.man
index 2398fbc..7c4ce58 100644
--- a/man/XSetWMProperties.man
+++ b/man/XSetWMProperties.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetWMProperties 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetWMProperties __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetWMProperties, XmbSetWMProperties, Xutf8SetWMProperties \- set standard window properties
 .SH SYNTAX
diff --git a/man/XSetWMProtocols.man b/man/XSetWMProtocols.man
index 44bf5ac..d75f408 100644
--- a/man/XSetWMProtocols.man
+++ b/man/XSetWMProtocols.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSetWMProtocols 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSetWMProtocols __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSetWMProtocols, XGetWMProtocols \- set or read a window's WM_PROTOCOLS property
 .SH SYNTAX
diff --git a/man/XStoreBytes.man b/man/XStoreBytes.man
index 40f4865..dda24e0 100644
--- a/man/XStoreBytes.man
+++ b/man/XStoreBytes.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XStoreBytes 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XStoreBytes __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XStoreBytes, XStoreBuffer, XFetchBytes, XFetchBuffer, XRotateBuffers \- manipulate cut and paste buffers
 .SH SYNTAX
diff --git a/man/XStoreColors.man b/man/XStoreColors.man
index bb90582..6b516ad 100644
--- a/man/XStoreColors.man
+++ b/man/XStoreColors.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XStoreColors 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XStoreColors __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XStoreColors, XStoreColor, XStoreNamedColor \- set colors
 .SH SYNTAX
diff --git a/man/XStringListToTextProperty.man b/man/XStringListToTextProperty.man
index 02cc216..a921048 100644
--- a/man/XStringListToTextProperty.man
+++ b/man/XStringListToTextProperty.man
@@ -142,7 +142,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XStringListToTextProperty 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XStringListToTextProperty __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XStringListToTextProperty, XTextPropertyToStringList, XFreeStringList, XTextProperty \- convert string lists and text property structure
 .SH SYNTAX
diff --git a/man/XStringToKeysym.man b/man/XStringToKeysym.man
index 9c3a48e..511bd3a 100644
--- a/man/XStringToKeysym.man
+++ b/man/XStringToKeysym.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XStringToKeysym 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XStringToKeysym __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XStringToKeysym, XKeysymToString, XKeycodeToKeysym, XKeysymToKeycode, XConvertCase \- convert keysyms
 .SH SYNTAX
diff --git a/man/XSupportsLocale.man b/man/XSupportsLocale.man
index 167421e..85111e0 100644
--- a/man/XSupportsLocale.man
+++ b/man/XSupportsLocale.man
@@ -142,7 +142,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSupportsLocale 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSupportsLocale __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSupportsLocale, XSetLocaleModifiers \- determine locale support and configure locale modifiers
 .SH SYNTAX
diff --git a/man/XSynchronize.man b/man/XSynchronize.man
index 65ca2c1..404631e 100644
--- a/man/XSynchronize.man
+++ b/man/XSynchronize.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XSynchronize 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XSynchronize __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XSynchronize, XSetAfterFunction \- enable or disable synchronization 
 .SH SYNTAX
diff --git a/man/XTextExtents.man b/man/XTextExtents.man
index 96634c5..becf7f4 100644
--- a/man/XTextExtents.man
+++ b/man/XTextExtents.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XTextExtents 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XTextExtents __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XTextExtents, XTextExtents16, XQueryTextExtents, XQueryTextExtents16 \- compute or query text extents
 .SH SYNTAX
diff --git a/man/XTextWidth.man b/man/XTextWidth.man
index 149878f..2e2c794 100644
--- a/man/XTextWidth.man
+++ b/man/XTextWidth.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XTextWidth 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XTextWidth __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XTextWidth, XTextWidth16 \- compute text width
 .SH SYNTAX
diff --git a/man/XTranslateCoordinates.man b/man/XTranslateCoordinates.man
index ab329a5..364636d 100644
--- a/man/XTranslateCoordinates.man
+++ b/man/XTranslateCoordinates.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XTranslateCoordinates 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XTranslateCoordinates __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XTranslateCoordinates \- translate window coordinates
 .SH SYNTAX
diff --git a/man/XUnmapEvent.man b/man/XUnmapEvent.man
index 261f086..c560a2a 100644
--- a/man/XUnmapEvent.man
+++ b/man/XUnmapEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XUnmapEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XUnmapEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XUnmapEvent \- UnmapNotify event structure
 .SH STRUCTURES
diff --git a/man/XUnmapWindow.man b/man/XUnmapWindow.man
index 84874ce..0c5f5aa 100644
--- a/man/XUnmapWindow.man
+++ b/man/XUnmapWindow.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XUnmapWindow 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XUnmapWindow __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XUnmapWindow, XUnmapSubwindows \- unmap windows
 .SH SYNTAX
diff --git a/man/XVaCreateNestedList.man b/man/XVaCreateNestedList.man
index 1ef3113..9017317 100644
--- a/man/XVaCreateNestedList.man
+++ b/man/XVaCreateNestedList.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XVaCreateNestedList 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XVaCreateNestedList __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XVaCreateNestedList \- allocate a nested variable argument list
 .SH SYNTAX
diff --git a/man/XVisibilityEvent.man b/man/XVisibilityEvent.man
index 503c034..0313aab 100644
--- a/man/XVisibilityEvent.man
+++ b/man/XVisibilityEvent.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XVisibilityEvent 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XVisibilityEvent __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XVisibilityEvent \- VisibilityNotify event structure
 .SH STRUCTURES
diff --git a/man/XWarpPointer.man b/man/XWarpPointer.man
index 2e88459..07ec5d5 100644
--- a/man/XWarpPointer.man
+++ b/man/XWarpPointer.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XWarpPointer 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XWarpPointer __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XWarpPointer \- move pointer
 .SH SYNTAX
diff --git a/man/XcmsAllocColor.man b/man/XcmsAllocColor.man
index c6fdbf7..bf93c73 100644
--- a/man/XcmsAllocColor.man
+++ b/man/XcmsAllocColor.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsAllocColor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsAllocColor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsAllocColor, XcmsAllocNamedColor \- allocate colors
 .SH SYNTAX
diff --git a/man/XcmsCCCOfColormap.man b/man/XcmsCCCOfColormap.man
index 9cee8e7..0303cd3 100644
--- a/man/XcmsCCCOfColormap.man
+++ b/man/XcmsCCCOfColormap.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsCCCOfColormap 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsCCCOfColormap __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsCCCOfColormap, XcmsSetCCCOfColormap \- query and modify CCC of a colormap
 .SH SYNTAX
diff --git a/man/XcmsCIELabQueryMaxC.man b/man/XcmsCIELabQueryMaxC.man
index abc80be..ef0c3ee 100644
--- a/man/XcmsCIELabQueryMaxC.man
+++ b/man/XcmsCIELabQueryMaxC.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsCIELabQueryMax 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsCIELabQueryMax __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsCIELabQueryMaxC, XcmsCIELabQueryMaxL, XcmsCIELabQueryMaxLC, XcmsCIELabQueryMinL \- obtain the CIE L*a*b* coordinates
 .SH SYNTAX
diff --git a/man/XcmsCIELuvQueryMaxC.man b/man/XcmsCIELuvQueryMaxC.man
index ee389d6..01f9f49 100644
--- a/man/XcmsCIELuvQueryMaxC.man
+++ b/man/XcmsCIELuvQueryMaxC.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsCIELuvQueryMaxC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsCIELuvQueryMaxC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsCIELuvQueryMaxC, XcmsCIELuvQueryMaxL, XcmsCIELuvQueryMaxLC, XcmsCIELuvQueryMinL \- obtain the CIE L*u*v* coordinates
 .SH SYNTAX
diff --git a/man/XcmsColor.man b/man/XcmsColor.man
index dcd7a7e..43635ce 100644
--- a/man/XcmsColor.man
+++ b/man/XcmsColor.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsColor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsColor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsColor, XcmsRGB, XcmsRGBi, XcmsCIEXYZ, XcmsCIEuvY, XcmsCIExyY, XcmsCIELab, XcmsCIELuv, XcmsTekHVC, XcmsPad \- Xcms color struture
 .SH STRUCTURES
diff --git a/man/XcmsConvertColors.man b/man/XcmsConvertColors.man
index e80986d..9e3ca61 100644
--- a/man/XcmsConvertColors.man
+++ b/man/XcmsConvertColors.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsConvertColors 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsConvertColors __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsConvertColors \- convert CCC color specifications
 .SH SYNTAX
diff --git a/man/XcmsCreateCCC.man b/man/XcmsCreateCCC.man
index ccf1548..24fcee8 100644
--- a/man/XcmsCreateCCC.man
+++ b/man/XcmsCreateCCC.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsCreateCCC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsCreateCCC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsCreateCCC, XcmsFreeCCC \- creating and destroying CCCs
 .SH SYNTAX
diff --git a/man/XcmsDefaultCCC.man b/man/XcmsDefaultCCC.man
index febe50e..fe703c1 100644
--- a/man/XcmsDefaultCCC.man
+++ b/man/XcmsDefaultCCC.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsDefaultCCC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsDefaultCCC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsDefaultCCC \- obtain the default CCC for a screen
 .SH SYNTAX
diff --git a/man/XcmsQueryBlack.man b/man/XcmsQueryBlack.man
index 96588c9..de67a6e 100644
--- a/man/XcmsQueryBlack.man
+++ b/man/XcmsQueryBlack.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsQueryBlack 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsQueryBlack __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsQueryBlack, XcmsQueryBlue, XcmsQueryGreen, XcmsQueryRed, XcmsQueryWhite \- obtain black, blue, green, red, and white CCC color specifications
 .SH SYNTAX
diff --git a/man/XcmsQueryColor.man b/man/XcmsQueryColor.man
index b485bef..b32b8c6 100644
--- a/man/XcmsQueryColor.man
+++ b/man/XcmsQueryColor.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsQueryColor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsQueryColor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsQueryColor, XcmsQueryColors, XcmsLookupColor \- obtain color values
 .SH SYNTAX
diff --git a/man/XcmsSetWhitePoint.man b/man/XcmsSetWhitePoint.man
index dca7069..1af22e2 100644
--- a/man/XcmsSetWhitePoint.man
+++ b/man/XcmsSetWhitePoint.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsSetWhitePoint 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsSetWhitePoint __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsSetWhitePoint, XcmsSetWhiteAdjustProc \- modifying CCC attributes
 .SH SYNTAX
diff --git a/man/XcmsStoreColor.man b/man/XcmsStoreColor.man
index 3630ad0..b51889b 100644
--- a/man/XcmsStoreColor.man
+++ b/man/XcmsStoreColor.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsStoreColor 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsStoreColor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsStoreColor, XcmsStoreColors \- set colors
 .SH SYNTAX
diff --git a/man/XcmsTekHVCQueryMaxC.man b/man/XcmsTekHVCQueryMaxC.man
index 28e7a4c..e98de02 100644
--- a/man/XcmsTekHVCQueryMaxC.man
+++ b/man/XcmsTekHVCQueryMaxC.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XcmsTekHVCQueryMaxC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XcmsTekHVCQueryMaxC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsTekHVCQueryMaxC, XcmsTekHVCQueryMaxV, XcmsTekHVCQueryMaxVC, XcmsTekHVCQueryMaxVSamples, XcmsTekHVCQueryMinV \- obtain the TekHVC coordinates
 .SH SYNTAX
diff --git a/man/XmbDrawImageString.man b/man/XmbDrawImageString.man
index e47c033..335077f 100644
--- a/man/XmbDrawImageString.man
+++ b/man/XmbDrawImageString.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XmbDrawImageString 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbDrawImageString __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbDrawImageString, XwcDrawImageString, Xutf8DrawImageString \- draw image text using a single font set
 .SH SYNTAX
diff --git a/man/XmbDrawString.man b/man/XmbDrawString.man
index 8af12ed..0af25f1 100644
--- a/man/XmbDrawString.man
+++ b/man/XmbDrawString.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XmbDrawString 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbDrawString __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbDrawString, XwcDrawString, Xutf8DrawString \- draw text using a single font set
 .SH SYNTAX
diff --git a/man/XmbDrawText.man b/man/XmbDrawText.man
index 3baf722..30fe5ea 100644
--- a/man/XmbDrawText.man
+++ b/man/XmbDrawText.man
@@ -142,7 +142,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XmbDrawText 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbDrawText __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbDrawText, XwcDrawText, Xutf8DrawText \- draw text using multiple font sets 
 .SH SYNTAX
diff --git a/man/XmbLookupString.man b/man/XmbLookupString.man
index fc30f4a..c2413ae 100644
--- a/man/XmbLookupString.man
+++ b/man/XmbLookupString.man
@@ -143,7 +143,7 @@
 ..
 .ny0
 '\" t
-.TH XmbLookupString 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbLookupString __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbLookupString, XwcLookupString, Xutf8LookupString \- obtain composed input from an input method
 .SH SYNTAX
diff --git a/man/XmbResetIC.man b/man/XmbResetIC.man
index e71465b..7939214 100644
--- a/man/XmbResetIC.man
+++ b/man/XmbResetIC.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XmbResetIC 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbResetIC __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbResetIC, XwcResetIC, Xutf8ResetIC \- reset the state of an input context
 .SH SYNTAX
diff --git a/man/XmbTextEscapement.man b/man/XmbTextEscapement.man
index 8aeb0b0..85c44d6 100644
--- a/man/XmbTextEscapement.man
+++ b/man/XmbTextEscapement.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XmbTextEscapement 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbTextEscapement __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbTextEscapement, XwcTextEscapement, Xutf8TextEscapement \- obtain the escapement of text
 .SH SYNTAX
diff --git a/man/XmbTextExtents.man b/man/XmbTextExtents.man
index 4c9ad9d..7275224 100644
--- a/man/XmbTextExtents.man
+++ b/man/XmbTextExtents.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XmbTextExtents 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbTextExtents __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbTextExtents, XwcTextExtents, Xutf8TextExtents \- compute text extents
 .SH SYNTAX
diff --git a/man/XmbTextListToTextProperty.man b/man/XmbTextListToTextProperty.man
index 1336737..cad0cf3 100644
--- a/man/XmbTextListToTextProperty.man
+++ b/man/XmbTextListToTextProperty.man
@@ -144,7 +144,7 @@
 ..
 .ny0
 '\" t
-.TH XmbTextListToTextProperty 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbTextListToTextProperty __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbTextListToTextProperty, XwcTextListToTextProperty, Xutf8TextListToTextProperty, XmbTextPropertyToTextList, XwcTextPropertyToTextList, Xutf8TextPropertyToTextList, XwcFreeStringList, XDefaultString \- convert text lists and text property structures
 .SH SYNTAX
diff --git a/man/XmbTextPerCharExtents.man b/man/XmbTextPerCharExtents.man
index ba1d2e3..3f7b2c4 100644
--- a/man/XmbTextPerCharExtents.man
+++ b/man/XmbTextPerCharExtents.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XmbTextPerCharExtents 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XmbTextPerCharExtents __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XmbTextPerCharExtents, XwcTextPerCharExtents, Xutf8TextPerCharExtents \- obtain per-character information for a text string
 .SH SYNTAX
diff --git a/man/XrmEnumerateDatabase.man b/man/XrmEnumerateDatabase.man
index 0da6490..67bc1e0 100644
--- a/man/XrmEnumerateDatabase.man
+++ b/man/XrmEnumerateDatabase.man
@@ -142,7 +142,7 @@
 ..
 .ny0
 '\" t
-.TH XrmEnumerateDatabase 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XrmEnumerateDatabase __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmEnumerateDatabase \- enumerate resource database entries
 .SH SYNTAX
diff --git a/man/XrmGetFileDatabase.man b/man/XrmGetFileDatabase.man
index 925eca9..959603a 100644
--- a/man/XrmGetFileDatabase.man
+++ b/man/XrmGetFileDatabase.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XrmGetFileDatabase 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XrmGetFileDatabase __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmGetFileDatabase, XrmPutFileDatabase, XrmGetStringDatabase, XrmLocaleOfDatabase, XrmGetDatabase, XrmSetDatabase, XrmDestroyDatabase \- retrieve and store resource databases
 .SH SYNTAX
diff --git a/man/XrmGetResource.man b/man/XrmGetResource.man
index 98e96dc..06d467c 100644
--- a/man/XrmGetResource.man
+++ b/man/XrmGetResource.man
@@ -142,7 +142,7 @@
 ..
 .ny0
 '\" e
-.TH XrmGetResource 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XrmGetResource __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmGetResource, XrmQGetResource, XrmQGetSearchList, XrmQGetSearchResource \- retrieve database resources and search lists
 .SH SYNTAX
diff --git a/man/XrmInitialize.man b/man/XrmInitialize.man
index 90484bf..c282e61 100644
--- a/man/XrmInitialize.man
+++ b/man/XrmInitialize.man
@@ -141,7 +141,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XrmInitialize 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XrmInitialize __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmInitialize, XrmParseCommand, XrmValue, XrmOptionKind, XrmOptionDescRec \- initialize the Resource Manager, Resource Manager structures, and parse the command line
 .SH SYNTAX
diff --git a/man/XrmMergeDatabases.man b/man/XrmMergeDatabases.man
index 835dfaf..22fc8f4 100644
--- a/man/XrmMergeDatabases.man
+++ b/man/XrmMergeDatabases.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XrmMergeDatabases 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XrmMergeDatabases __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmMergeDatabases, XrmCombineDatabase, XrmCombineFileDatabase \- merge resource databases
 .SH SYNTAX
diff --git a/man/XrmPutResource.man b/man/XrmPutResource.man
index 80c59cb..e42fbf8 100644
--- a/man/XrmPutResource.man
+++ b/man/XrmPutResource.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XrmPutResource 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XrmPutResource __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmPutResource, XrmQPutResource, XrmPutStringResource, XrmQPutStringResource, XrmPutLineResource \- store database resources
 .SH SYNTAX
diff --git a/man/XrmUniqueQuark.man b/man/XrmUniqueQuark.man
index 45e5348..26332be 100644
--- a/man/XrmUniqueQuark.man
+++ b/man/XrmUniqueQuark.man
@@ -140,7 +140,7 @@
 .el .sp 10p
 ..
 .ny0
-.TH XrmUniqueQuark 3X11 __xorgversion__ "XLIB FUNCTIONS"
+.TH XrmUniqueQuark __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmUniqueQuark, XrmStringToQuark, XrmPermStringToQuark, XrmQuarkToString, XrmStringToQuarkList, XrmStringToBindingQuarkList \- manipulate resource quarks
 .SH SYNTAX
