Index: libx11/src/xlibi18n/lcFile.c
===================================================================
--- libx11.orig/src/xlibi18n/lcFile.c	2006-06-30 02:34:10.000000000 -0400
+++ libx11/src/xlibi18n/lcFile.c	2006-06-30 02:35:06.000000000 -0400
@@ -269,7 +269,11 @@
 	    if (seteuid(0) != 0) {
 		priv = 0;
 	    } else {
-		seteuid(oldeuid);
+        if (seteuid(oldeuid) == -1) {
+            /* XXX ouch, coudn't get back to original uid 
+             what can we do ??? */
+            _exit(127);
+        }
 		priv = 1;
 	    }
 #endif
