--- libx11.orig/nls/en_US.UTF-8/Compose.pre
+++ libx11/nls/en_US.UTF-8/Compose.pre
@@ -6176,3 +6176,13 @@
 ##<Multi_key> <U1D1B9> <U1D165> <U1D16F> 	: "𝆹𝅥𝅯"   U1D1BF # MUSICAL SYMBOL FUSA WHITE
 <Multi_key> <U1D1BC> <U1D16F> 	: "𝆺𝅥𝅯"   U1D1C0 # MUSICAL SYMBOL FUSA BLACK
 ##<Multi_key> <U1D1BA> <U1D165> <U1D16F> 	: "𝆺𝅥𝅯"   U1D1C0 # MUSICAL SYMBOL FUSA BLACK
+
+XCOMM Khmer digraphs
+XCOMM A keystroke has to generate several characters, so they are defined
+XCOMM in this file
+
+<U17fb>    :   "ុះ"
+<U17fc>    :   "ុំ"
+<U17fd>    :   "េះ"
+<U17fe>    :   "ោះ"
+<U17ff>    :   "ាំ"
