Stolen from HEAD, commit d6fba1f44d404362d3be1b421f57d7ccc3c8cdac
Closes: #387133

Index: libx11/src/util/makekeys.c
===================================================================
--- libx11.orig/src/util/makekeys.c
+++ libx11/src/util/makekeys.c
@@ -1,5 +1,5 @@
 /* $Xorg: makekeys.c,v 1.5 2001/02/09 02:03:40 $ */
-/* $XdotOrg: xc/lib/X11/util/makekeys.c,v 1.4 2004/09/26 20:46:17 kuhn Exp $ */
+/* $XdotOrg: lib/X11/src/util/makekeys.c,v 1.5 2005-07-03 07:00:56 daniels Exp $ */
 /*
 
 Copyright 1990, 1998  The Open Group
@@ -49,7 +49,7 @@
     KeySym	val;
 } info[KTNUM];
 
-#define MIN_REHASH 10
+#define MIN_REHASH 15
 #define MATCHES 10
 
 char tab[KTNUM];
