Index: libx11/acinclude.m4
===================================================================
--- libx11.orig/acinclude.m4	2006-10-09 12:30:05.000000000 -0400
+++ libx11/acinclude.m4	2006-10-10 21:03:19.000000000 -0400
@@ -4,7 +4,10 @@
   exec_prefix_NONE=
   test "x$prefix" = xNONE && prefix_NONE=yes && prefix=$ac_default_prefix
   test "x$exec_prefix" = xNONE && exec_prefix_NONE=yes && exec_prefix=$prefix
+dnl In Autoconf 2.60, ${datadir} refers to ${datarootdir}, which in turn
+dnl refers to ${prefix}.  Thus we have to use `eval' twice.
   eval ac_define_dir="\"[$]$2\""
+  eval ac_define_dir="\"$ac_define_dir\""
   AC_SUBST($1, "$ac_define_dir")
   AC_DEFINE_UNQUOTED($1, "$ac_define_dir", [$3])
   test "$prefix_NONE" && prefix=NONE
Index: libx11/configure.ac
===================================================================
--- libx11.orig/configure.ac	2006-10-10 21:02:51.000000000 -0400
+++ libx11/configure.ac	2006-10-10 21:03:19.000000000 -0400
@@ -307,7 +307,12 @@
 fi
 
 X11_DATADIR="${datadir}/X11"
+AC_DEFINE_DIR(X11_DATADIR, X11_DATADIR, [Location of libX11 data])
+AC_SUBST(X11_DATADIR)
+
 X11_LIBDIR="${libdir}/X11"
+AC_DEFINE_DIR(X11_LIBDIR, X11_LIBDIR, [Location of libX11 library data])
+AC_SUBST(X11_LIBDIR)
 
 X11_CFLAGS="$X11_CFLAGS $XPROTO_CFLAGS"
 AC_SUBST(X11_CFLAGS)
@@ -322,20 +327,23 @@
 #
 
 X11_LOCALEDATADIR="${X11_DATADIR}/locale"
+AC_DEFINE_DIR(XLOCALEDATADIR, X11_LOCALEDATADIR, [Location of libX11 locale data])
+AC_SUBST(X11_LOCALEDATADIR)
+
 X11_LOCALELIBDIR="${X11_LIBDIR}/locale"
+AC_DEFINE_DIR(XLOCALELIBDIR, X11_LOCALELIBDIR, [Location of libX11 locale libraries])
+AC_SUBST(X11_LOCALELIBDIR)
+
 X11_LOCALEDIR="${X11_LOCALEDATADIR}"
-XKEYSYMDB="${X11_DATADIR}/XKeysymDB"
-XERRORDB="${X11_DATADIR}/XErrorDB"
+AC_DEFINE_DIR(XLOCALEDIR, X11_LOCALEDIR, [Location of libX11 locale data])
+AC_SUBST(X11_LOCALEDIR)
 
-AC_SUBST(X11_DATADIR)
-AC_SUBST(X11_LIBDIR)
-AC_SUBST(X11_LOCALEDATADIR)
-AC_SUBST(X11_LOCALELIBDIR)
-AC_DEFINE_DIR(X11_DATADIR, X11_DATADIR, [Location of libX11 data])
+XKEYSYMDB="${X11_DATADIR}/XKeysymDB"
 AC_DEFINE_DIR(XKEYSYMDB, XKEYSYMDB, [Location of keysym database])
+
+XERRORDB="${X11_DATADIR}/XErrorDB"
 AC_DEFINE_DIR(XERRORDB, XERRORDB, [Location of error message database])
-AC_DEFINE_DIR(XLOCALEDIR, X11_LOCALEDIR, [Location of libX11 locale data])
-AC_DEFINE_DIR(XLOCALELIBDIR, X11_LOCALELIBDIR, [Location of libX11 locale libraries])
+
 
 XORG_CHECK_MALLOC_ZERO
 XORG_RELEASE_VERSION
